### PR TITLE
Add BCD for CSS scale, rotate, and translate properties

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1207734

### CODE REVIEW

File: css/properties/rotate.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "x_y_z": {
+          "__compat": {
+            "description": "Support for x, y, and z axis values",
```
Comment (teoli2003) [inline css/properties/rotate.json]: It is not clear what this relate too on the MDN page.

What about "Three length/percentage values" ? Shorter and match the text on the page.

**Comment by chrisdavidmills** [comment]
I've looked at this, updated the wording on the page slightly (see https://developer.mozilla.org/en-US/docs/Web/CSS/rotate#Values), and updated this wording to match it.

File: css/properties/rotate.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
+  "css": {
+    "properties": {
+      "translate": {
```
Comment (teoli2003) [inline css/properties/rotate.json]: "rotate"

File: css/properties/rotate.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/translate",
```
Comment (teoli2003) [inline css/properties/rotate.json]: "rotate"

File: css/properties/rotate.json [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "x_y_z": {
```
Comment (teoli2003) [inline css/properties/rotate.json]: "x_y_z_angle"

File: css/properties/rotate.json [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+          "__compat": {
+            "description": "Support for x, y, or z axis name plus angle value",
```
Comment (teoli2003) [inline css/properties/rotate.json]: Avoid support (it is implicit): "x, y, or z axis name plus angle value" or "Axis name (x, y, or z) plus angle value"

**Comment by chrisdavidmills** [comment]
Oops, sorry for the stupid copy and paste error - rotate is really rotate now ;-)

Fixed the other bits too.

**Pull request merged**